docs/Marketplace.md

```diff
@@ -338,62 +412,62 @@ curl 'http://localhost:8000/api/codex/v1/sales/availability' \
 ```
 
-Once the upload completes, you should see a CID (e.g. `zDvZRwzm2mK7tvDzKScRLapqGdgNTLyyEBvx1TQY37J2CdWdS6Sj`) for the file printed to the terminal. Use that CID in the purchase request:
+Once the upload completes, you should see a _Content Identifier_, or _CID_ (e.g. `zDvZRwzm2mK7tvDzKScRLapqGdgNTLyyEBvx1TQY37J2CdWdS6Sj`) for the uploaded file printed to the terminal. Use that CID in the purchase request:
 
 ```bash
+# make sure to replace the CID before with the CID you got in the previous step
 export CID=zDvZRwzm2mK7tvDzKScRLapqGdgNTLyyEBvx1TQY37J2CdWdS6Sj
-export EXPIRY_TIME=$((1000 + $(date +%s))) # current time + 1000 seconds
-                                           # adjust expiry_time as desired, see below
 ```
 
 ```bash
 curl "http://localhost:8001/api/codex/v1/storage/request/${CID}" \
   --header 'Content-Type: application/json' \
   --data "{
-    \"duration\": \"1200\",
+    \"duration\": \"600\",
     \"reward\": \"1\",
     \"proofProbability\": \"3\",
-    \"expiry\": \"${EXPIRY_TIME}\",
-    \"nodes\": 1,
-    \"tolerance\": 0,
+    \"expiry\": \"500\",
+    \"nodes\": 3,
+    \"tolerance\": 1,
     \"collateral\": \"1000\"
   }"
 ```
 
 The parameters under `--data` say that:
 
-1. we want to purchase storage for our file for $20$ minutes (`"duration": "1200"`);
-2. we are willing to pay up to $1$ token per byte, per second (`"reward": "1"`);
-3. our file will be split into four pieces (`"nodes": 3` and `"tolerance": 1`), so that we only need three pieces to rebuild the file; i.e., we can tolerate that at most one node stops storing our data; either due to failure or other reasons;
-4. we demand `1000` tokens in collateral from storage providers for each piece. Since there are $4$ such pieces, there will be `4000` in total collateral committed by all of the storage providers taken together once our request is fulfilled.
-
-Finally, the `expiry` puts a cap on the block time at which our request expires. This has to be at most `current block time + duration`, which means this request can fail if you input the wrong number, which you likely will if you do not know what the current block time is. Fear not, however, as you can try an an arbitrary number (e.g. `1000`), and look at the failure message:
-
-   `Expiry needs to be in future. Now: 1711995463`
-
-to compute a valid one. Just take the number in the error message and add the duration; i.e., `1711995463 + 1200 = 1711996663`, then use the resulting number (`1711996663`) as expiry and things should work. The request should return a purchase ID (e.g. `1d0ec5261e3364f8b9d1cf70324d70af21a9b5dccba380b24eb68b4762249185`), which you can use track the completion of your request in the marketplace.
+1. we want to purchase storage for our file for $10$ minutes (`"duration": "1200"`);
+2. we are willing to pay up to $1$ token per slot per second (`"reward": "1"`) (**MC:** I think `per second per byte` is still not implemented - this my first assignment, right?");
```

Comment:
I guess you'd know this better than I do. :slightly_smiling_face:

Reply:
indeed, I think I should...